WcaOnRails/config/locales/en.yml

```diff
@@ -1440,6 +1440,10 @@ en:
       wqac:
         name: "WCA Quality Assurance Committee"
         description: "This committee has the role to support and promote continuous quality improvement within the WCA, as well as worldwide application of quality standards to ensure consistent high quality of processes, WCA Staff, Regional Organizations, Competition Organizers, and Competition Staff."
+      #context: WCAT
+      wcat:
+        name: "WCA Competition Announcement Team"
+        description: "This committee has the role to approve and announce WCA Competitions and ensure such announcements adhere to WCA quality standards."
```

Comment:
Almost everything else is either a committee or a team. Here we use the word "team" in the title, but "committee" in the description, which is a bit confusing.

Reply:
I agree. @cubewhiz, because this text came from him, which I believe he copied from elsewhere.

Reply:
Committee and team can be used interchangeably, but feel free to use "team" instead for consistency.